src/plot_api/plot_config.js

```diff
@@ -471,7 +471,21 @@ var configAttributes = {
             'instead of MM/DD/YYYY). Currently `grouping` and `currency` are ignored',
             'for our automatic number formatting, but can be used in custom formats.'
         ].join(' ')
-    }
+    },
+
+    accessibility: {
+        valType: 'any',
+        dflt: {
+            library: 'chart2music',
+            enabled: true,
+            options: {},
+            info: {},
+            closedCaptions: {generate: false, elId: 'c2m-plotly-cc', elClassname: 'c2m-plotly-closed_captions'}
```

Comment:
I feel like all of these options are kind of overkill. Why do we need to specify both id and classname? And why are we making it an option that _we_ generate the closed caption div? I think we should just simplify this into one field called `closedCaptionsElId` and just assume users will make that div. I don't want to overcomplicate this

Reply:
Looking into this more I actually think we can just keep these in.